roles/katello_devel/meta/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+vars:
+  foreman_installer_options_internal_use_only:
+    - "--disable-system-checks"
```

Comment:
quotes unnecessary

Reply:
Done in 66ca393f53ff85915206cce67206f80425cd35f9